docs-source/content/userguide/managing-domains/domain-lifecycle/restarting.md

```diff
@@ -35,7 +35,6 @@ For detailed information on how to restart servers in a Oracle WebLogic Server i
 This document describes what actions you need to take to properly restart your servers for a number of common scenarios:
 
 * Modifying the WebLogic configuration
-* Changing the custom domain configuration overrides (also called situational configuration) for Domain in PV and Domain in Image domains
```

Comment:
Doesn't this still belong? There are still cases where a change may require a restart.

Reply:
I guess if the distribution strategy is ON_ROLLING.